pkg/genericapiserver/genericapiserver.go

```diff
@@ -196,33 +197,17 @@ type APIGroupInfo struct {
 
 // Config is a structure used to configure a GenericAPIServer.
 type Config struct {
+	genericapiservercfg.APIServerConfig
```

Comment:
Can you add comments for someone adding a new param. When should the new param go in APIServerConfig and when should it go in InternalConfig?